diktat-maven-plugin/src/test/kotlin/org/cqfn/diktat/plugin/maven/DiktatMavenPluginIntegrationTest.kt

```diff
@@ -5,60 +5,105 @@ import com.soebes.itf.jupiter.extension.MavenJupiterExtension
 import com.soebes.itf.jupiter.extension.MavenTest
 import com.soebes.itf.jupiter.maven.MavenExecutionResult
 import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.TestInfo
 import java.io.File
 import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.Path
+import kotlin.io.path.deleteIfExists
+import kotlin.io.path.div
 import kotlin.io.path.readText
 
 /**
  * Integration tests for diktat-maven-plugin. Run against the project from diktat-examples.
  * The whole pipeline is as follows:
- * * For each test case, test data is copied from examples with respect to maven-itf requirements, .mvn/jvm.config and .mvn/maven.config are copied too
- *   Note: for maven itf test name should equal example project's directory name, which we have in pom.xml.
- * * maven-failsafe-plugin launches tests; for each test case a separate maven process is spawned. diktat.version is taken from .mvn/maven.config
+ * * For each test case, test data is copied from examples with respect to maven-itf requirements, .mvn/jvm.config and .mvn/maven.config are copied, too.
+ *   **Note**: for maven its test name should equal example project's directory name, which we have in `pom.xml`.
```

Comment:
```suggestion
 *   **Note**: for maven-itf-plugin, test name should equal example project's directory name, which we have in `pom.xml`.
```
I think I've meant maven-itf-plugin here :)

Reply:
Thanks!